src/static/js/ace2_inner.js

```diff
@@ -3374,7 +3374,7 @@ function Ace2Inner(){
       {
         try
         {
-          var newWindow = window.open(n.href, '_blank');
+          var newWindow = window.open(n.href, '_blank', 'noopener, noreferrer');
```

Comment:
According to the [MDN documentation about `window.open()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#Window_functionality_features) newWindow will be null after this call, because of `noopener`.

> ## noopener
> If this feature is set, the newly-opened window [...] will not have access back to the originating window (via `Window.opener` — it returns `null`). In addition, the `window.open()` call will also return `null`, so the originating window will not have access to the new one either. [...]